src/pages/createInvoice.jsx

```diff
@@ -1,85 +1,185 @@
 import {useState} from 'react';
 
+import StyledButton from '../components/Button/styled';
+import Header from '../components/Header';
+import InputCard from '../components/Positioning/InputCard';
+import Wrapper from '../components/Positioning/Wrapper';
+
 export default function CreateInvoice() {
 	const [invoice, setInvoice] = useState(false);
 	/* -------------------USER DATA ---------------------*/
 	const [name, setName] = useState('');
 	const [street, setStreet] = useState('');
 	const [city, setCity] = useState('');
 	const [taxID, setTaxID] = useState('');
+	/* ________________USER NAME_________________________ */
+	const [recipientName, setRecipientName] = useState('');
+	const [recipientStreet, setRecipientStreet] = useState('');
+	const [recipientCity, setRecipientCity] = useState('');
+	/* ______________________ITEM/SERVICE______________________ */
+	const [service, setService] = useState('');
 	const [amount, setAmount] = useState('');
+	const [quantity, setQuantity] = useState('');
+	/* ______________Calculating from quantity and price________ */
+	const subTotal = amount * quantity;
+	const VAT = subTotal * 0.19;
+	const grandTotal = subTotal + VAT;
+
+	/* const subtotal
+	const grandtotal
+	*/
 
 	return (
 		<div>
 			{invoice ? (
 				<div>
-					<h1>Your Invoice</h1>
-					<ul>
-						<li>{name}</li>
-						<li>{street}</li>
-						<li>{city}</li>
-						<li>{taxID}</li>
-						<div>
-							<li>
-								<h1>{amount}</h1>
-							</li>
-						</div>
-					</ul>
-					<button onClick={() => setInvoice(false)}>Edit</button>
+					<Wrapper>
+						<h4>Logo</h4>
+						<h4>InvoiceNr.</h4>
+					</Wrapper>
+					<Wrapper>
+						<InputCard>
+							<li>Name: {name}</li>
+							<li>Adress: {street}</li>
+							<li>City: {city}</li>
+							<li>TAX ID: {taxID}</li>
+						</InputCard>
+						<InputCard>
+							<li>Name:{recipientName}</li>
+							<li>Adress:{recipientStreet}</li>
+							<li>City:{recipientCity}</li>
+						</InputCard>
+					</Wrapper>
+					<Wrapper>
+						<div>Service:</div>
+
+						<div>price (net):</div>
+
+						<div>QTY:</div>
```

Comment:
```suggestion
						<div>Service:</div>

						<div>price (net):</div>

						<div>QTY:</div>
```
try to find better semantic elements here